### DISCUSSION TITLE
Why no '-startdate' / '-enddate' parameters in 'openssl req'

### DISCUSSION THREAD

**Discussion by swaeberle** [openssl/openssl#21693] [General Discussion]
Is there any reason why `openssl req` does not accept parameters `-startdate` and `-enddate` like `openssl ca` when creating a self-signed certificate?

**Comment by t8m** [comment]
Yes, there is a reason, nobody implemented that feature. The same would be true for the x509 command which can also create certificates.

**Reply by swaeberle** [reply]
Okay, this sounds like someone should send a pull request ;)
I think I'll have a look at it as I just stumbled upon this and it looks not too complex (at least for `req`, `x509` seems to have some side-effects).

**Reply by paulidale** [reply]
Definitely, a pull request would be welcomed.
